mealplanner-ui/src/pages/MealPlans/MealPlans.tsx

```diff
@@ -169,7 +169,7 @@ const MealPlanCard = (props: MealPlanCardProps) => {
           </Typography>
         </CardContent>
         <CardActions disableSpacing>
-          <IconButton aria-label="add to favorites">
+          <IconButton aria-label="add to favorites" sx={{ "& :hover": { color: "#dc143c" } }}>
```

Comment:
Need to add the color from the theme -> secondary -> dark. You can access it with palette.